### PR TITLE
Don't pass Rterm comments

### PR DESCRIPTION
**Pull request by Ladvien**

### CODE REVIEW

**Comment by Ikuyadeu** [comment]
@Ladvien Thank you for your Pull Request again!
This PR looks extends #13. So I'll merge if that merged.

**Comment by Ikuyadeu** [comment]
@Ladvien This is conflicted, could you solve this?

File: src/extension.ts [hunk 1]
```diff
@@ -83,12 +83,17 @@ export function activate(context: ExtensionContext) {
         let { start, end } = window.activeTextEditor.selection;
         let currentDocument = window.activeTextEditor.document;
         let range = new Range(start, end);
-        const selectedLineText = !range.isEmpty
+        var selectedLineText = !range.isEmpty
```
Comment (Ikuyadeu) [inline src/extension.ts]: `let` is better

File: src/extension.ts [hunk 2]
```diff
@@ -100,6 +105,15 @@ export function activate(context: ExtensionContext) {
         }
     }
 
+    function checkForComment(string): boolean {
```
Comment (Ikuyadeu) [inline src/extension.ts]: variable name `string` should be change like `str` or `line`

File: src/extension.ts [hunk 3]
```diff
@@ -100,6 +105,15 @@ export function activate(context: ExtensionContext) {
...
+        var index = 0;
+        while(index < string.length){
+            if(!(string[index] == ' ')){ break };
+            index++;
+        }
+        if(string[index] == '#'){ return true } else { return false };
+    }
```
Comment (Ikuyadeu) [inline src/extension.ts]: return line[index] === '#'; is better.

File: src/extension.ts (hunk 1 above)
Comment (Ladvien) [inline src/extension.ts]: Done on all.

**Comment by Ikuyadeu** [comment]
@Ladvien Good! Thank you!
And, I'll publish this in new version within two days.

**Comment by Ladvien** [comment]
No worries.  I'm trying to hack away at creating a dataframe viewer. 

Thanks again for the ext! 👍

**Comment by Ikuyadeu** [comment]
Great!

**Pull request merged**